pom.xml

```diff
@@ -2536,6 +2536,8 @@
                 <Xmaxerrs>10000</Xmaxerrs>
                 <Xmaxwarns>10000</Xmaxwarns>
               </compilerArguments>
+              <compilerArgument>-Xlint:all</compilerArgument>
+              <compilerArgument>-Alint=trustArrayLenZero </compilerArgument>
```

Comment:
Try to put new lines before `</compilerArguments>`

Reply:
done!

Reply:
Not done

Reply:
```
<compilerArguments>
                <Xlint>all</Xlint>
...
```

Reply:
Nope It still aint work !

Reply:
Let's ask for help from checkerframefork team, please create issue or discussion in their repository

Reply:
Done : https://github.com/typetools/checker-framework/issues/6702

Reply:
![Screenshot from 2024-07-09 06-49-34](https://github.com/typetools/checker-framework/assets/96484809/a1823c54-7c47-4cb3-89ed-3236f910d4db)

![Screenshot from 2024-07-09 06-49-31](https://github.com/typetools/checker-framework/assets/96484809/6bfc3520-00d4-427c-88b9-c86925a39eaf)

![Screenshot from 2024-07-09 06-49-16](https://github.com/typetools/checker-framework/assets/96484809/170eaaa3-5cad-494c-bf90-1d125b8c8d8f)

![Screenshot from 2024-07-09 06-49-09](https://github.com/typetools/checker-framework/assets/96484809/79320dc7-3c78-4f5f-88a8-9a2f59fd06f1)

@romani  These are the results i found when I actually configured checker framework in one of the demo project. 

It says Unsupported Command line option : Alint=trustArrayLenZero

Reply:
Please share this test project in GitHub and show it to it maintainer, to let them suggest how to configure it.

Reply:
There's also `<compilerArgs combine.children="append">` try to put it there

Reply:
> There's also `<compilerArgs combine.children="append">` try to put it there

Its already there !